src/plone/restapi/tests/http-examples/relations_get_source.resp

```diff
@@ -0,0 +1,70 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "@id": "http://localhost:55001/plone/@relations?source=SomeUUID000000000000000000000001",
+    "items": {
+        "comprisesComponentPart": [
+            {
+                "source": {
```

Comment:
I discussed this with @ksuess. We will keep it as is for now, because it is useful to have a consistent format for the result items. If we need an optimization to remove the duplicated sources/targets in the future, we can add a parameter for it.